### ISSUE TITLE
Add python3 support

### ISSUE THREAD

**Issue by mts749** [reubano/csv2ofx#2]
I tried to install csv2ofx on a (pretty vanilla) OSX 10.11.2 system:

```
~> sudo pip install -v csv2ofx
Downloading/unpacking csv2ofx
  Could not find any downloads that satisfy the requirement csv2ofx
Cleaning up...
  Removing temporary dir /private/tmp/pip_build_root...
No distributions at all found for csv2ofx
Exception information:
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/pip-1.4.1-py2.7.egg/pip/basecommand.py", line 134, in main
    status = self.run(options, args)
  File "/Library/Python/2.7/site-packages/pip-1.4.1-py2.7.egg/pip/commands/install.py", line 236, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/Library/Python/2.7/site-packages/pip-1.4.1-py2.7.egg/pip/req.py", line 1085, in prepare_files
    url = finder.find_requirement(req_to_install, upgrade=self.upgrade)
  File "/Library/Python/2.7/site-packages/pip-1.4.1-py2.7.egg/pip/index.py", line 265, in find_requirement
    raise DistributionNotFound('No distributions at all found for %s' % req)
DistributionNotFound: No distributions at all found for csv2ofx
```

Any idea what's wrong?
Thanks.

**Comment by reubano** [comment]
Yea, I see the issue. I haven't yet gotten py3 support. Will do so soon.

**Comment by mts749** [comment]
Thanks. As far as I know I'm running python 2.7.10:
```
~> which python
/usr/bin/python
~> /usr/bin/python -V
Python 2.7.10
```
But I'm not really sure if this is definitive.

**Comment by reubano** [comment]
Hmmm that's odd. I installed into a new virtualenv and it worked fine. It was just the py3 virtualenv that failed for me.

```bash
reubano@tokpro [~]$ which python
/opt/local/bin/python
reubano@tokpro [~]$ python -V
Python 2.7.11
reubano@tokpro [~]$ pip --version
pip 7.1.2 from /opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages (python 2.7)
reubano@tokpro [~]$ which virtualenv
/opt/local/Library/Frameworks/Python.framework/Versions/2.7/bin/virtualenv
```

```bash
virtualenv ~/tmp
source ~/tmp/bin/activate
pip install csv2ofx
```

**Comment by mts749** [comment]
I'm not using virtualenv but my pip is different:
```
~> pip --version
pip 1.4.1 from /Library/Python/2.7/site-packages/pip-1.4.1-py2.7.egg (python 2.7)
```

**Comment by mts749** [comment]
However when I installed and ran virtualenv (as above since I have never used it) it worked,

**Comment by reubano** [comment]
I just installed gloablly as well 

```bash
reubano@tokpro [~]$ pip install csv2ofx
Collecting csv2ofx
  Downloading csv2ofx-0.18.2-py27-none-any.whl
...
reubano@tokpro [~]$ csv2ofx -V
v0.18.2
```

Can you try upgrading... first `pip` and if that doesn't work, then python?

**Comment by mts749** [comment]
OK will do.

**Comment by mts749** [comment]
Problem solved by upgrading to pip 7.1.2.
Many thanks!

**Comment by reubano** [comment]
Awesome!
